carraway.gemspec

```diff
@@ -27,7 +27,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'rack-flash3'
   spec.add_dependency 'redcarpet'
 
-  spec.add_development_dependency "bundler", "~> 1.16"
+  spec.add_development_dependency "bundler", "~> 2.0"
```

Comment:
Does this enhancement require bundler 2? If you want to just update, I think you should make another pull request.

Reply:
This enhancement doesn't require bundler 2. I'll make another pull request.